"make up" fails with error "start_period was unexpected"

Thank you for this great example :)

"make up" fails with error "start_period was unexpected" on Ubuntu 20.04 box:
docker version: 20.10.9
docker-compose version: 1.25.0

Here I attached is a log from "make up" command.

```
ubuntu@dev1:~/keycloak-vault$ make up
docker-compose -f docker-compose.yml up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.keycloak.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
services.vault.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
make: *** [Makefile:14: up] Error 1
```

on above environment, just changing compose file version,
version: '3.3' to version: '3.7' solves the problem.

ref.
https://github.com/docker/compose/issues/5177

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,4 +1,4 @@
-version: '3.3'
+version: '3.7'
 
 services:
 

